capeprivacy/cape
# thor/broker

- Move oauth from enteprise into coordinator
- Move detokenization from enterprise to here
- Docker changes for detokenization job launcher
- move ui embed from enterprise to here
- Added DB logic for schema reporting
- Pull over charts from enterprise
- add pull secrets to migrations job
- bump chart version
- cleanup migrations by removing policies and attachments tables
- remove the suggestions api
- Send label & version to set policy cmd
- fix cors on k8s deployment
- bump chart version
- Add missing dependencies to policy schema
- initial database work for organizations
- add org membership postgres database
- Stubs starting for signup
- signup flow implemented in the db
- add tests for the database components
- lint action
- docker action
- add graphql routes for orgs plus utility CLI commands
- cr comment
- rename migration
- envconfig needed an update for cors env var to work
- bump chart version
- added a setProjectStatus route
- added the route to verify a signup
- adds a verification middleware
- added the route to verify a signup
- unused const block
- can get version & label on policy in graphql
- use alice chain for gql routes
- add a project owner field, can be a user email or org name
- make issuer static so people can log in after restarts
- List owners (user or org) projects
- Adds the ability to query a users orgs
- give a user a role when they signup
- adding tasks to projects
- alternative testing strategy for gql
- updates to alternative testing strat
- YEEEEET
- assert on what is passed to the db
- List an organization's members
- Only returns projects that a user is a contributor
- Session improvements
- removes ModelLabel, ModelEmail and ModelRole
- dont deploy to cluster that is not kind-cape-local cluster
- db integration tests
- Add SMTP Mailer implementation
- Bump chart version
- Fix Auth identity and use snake case for log key
- passing tests
- make sure dbs are set on resolver
- add tests for db impl
- cleanup graphql for tasks a bit
- Add To, From, and Subject headers to email templates
- add way to get individual task and little bit of cleanup
- add task param on project in graphql
- Add username field to a user
- forgetting username migration
- Add an org creator to the orgs list of members
- enable ui for local development
- bump chart version
- Signup flow changes
- ci fixes
- ci
- rename tasks to computations for now
- add string computation field
- Add computation session model, db and schema
- reordered migrations so wasnt migrating properly, should fix
- Add initialize session implementation for choreo
- add computation resolver implementation
- Return sessions related to projects
- wip converting the audit log db to be project specific
- working project log
- fix gql req formatting
- project lint
- expose created_at via gql
- Added number of logs field on project
- CR
- added definitions for tasks
- added high level "cape tasks" to the app
- green build
- implemented single task get
- use db error
- wrap all errors
- lint
- Adds an in memory computation session mgr
- add get session status
- remove detokenization, update dockerfile
- remove python from actions
- add name field to a token
- Added an ID to events
- return task list in reverse chronological order
- can assign users to a task
- wip for justin
- multiplication task returns a real computation
- computation sessions reference tasks instead of computations
- finish up task roles, comp sessions impl
- add computation to tasks
- return json string computation
- wip for morten
- some more tests
- forgotten file tasks_test.go file
- more tests
- more tests
- fix up pr comments
- remove policy from cape
- returning base64 bytes for computation (#53)
- Add support for additional ingress endpoints to the helm chart
